client/signup/config/flows.js

```diff
@@ -3,6 +3,7 @@ import {
 	DOT_ORG_THEME,
 	WOOCOMMERCE_THEME,
 	MARKETPLACE_THEME,
+	// eslint-disable-next-line import/named
 	shouldGoToAssembler,
```

Comment:
@arthur791004 - Can you double check this import? eslint didn't like it. It _seems_ like the item is exported though.